packages/editor-ui/src/stores/workflowHistory.store.ts

```diff
@@ -40,9 +45,65 @@ export const useWorkflowHistoryStore = defineStore('workflowHistory', () => {
 			return null;
 		});
 
+	const downloadVersion = async (workflowId: string, workflowVersionId: WorkflowVersionId) => {
+		const [workflow, workflowVersion] = await Promise.all([
+			workflowsStore.fetchWorkflow(workflowId),
+			getWorkflowVersion(workflowId, workflowVersionId),
+		]);
+		if (workflowVersion?.nodes && workflowVersion?.connections && workflow) {
+			const { connections, nodes } = workflowVersion;
+			const blob = new Blob([JSON.stringify({ ...workflow, nodes, connections }, null, 2)], {
+				type: 'application/json;charset=utf-8',
+			});
+			saveAs(blob, `${workflow.name.replace(/[^a-zA-Z0-9]/gi, '_')}-${workflowVersionId}.json`);
```

Comment:
Why are we so strict about filename chars? I'd expect non-ASCII chars should be allowed when saving to the filesystem? Non-English speakers are likely to use them when naming a workflow.

Reply:
I'm fine with downloading the file with the workflow name as is, updated